d2renderers/d2svg/d2svg.go

```diff
@@ -958,6 +1080,39 @@ func drawShape(writer io.Writer, targetShape d2target.Shape, sketchRunner *d2ske
 			}
 		}
 	case d2target.ShapeText, d2target.ShapeCode:
+	case d2target.ShapeHexagon:
```

Comment:
can u put this above the line above? i think slightly nicer to have the "do nothing" cases next to default and not sandwiched between the "do something" cases

Reply:
Alright! I've changed the order